include/secp256k1_musig.h

```diff
@@ -223,6 +223,24 @@ SECP256K1_API int secp256k1_musig_pubkey_agg(
     size_t n_pubkeys
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(5);
 
+/** Obtain the aggregate public key from a keyagg_cache.
+ *
+ *  This is only useful if you need the non-xonly public key, in particular for
+ *  ordinary (non-xonly) tweaking or batch-verifying multiple key aggregations
+ *  (not implemented).
```

Comment:
(not this PR:) We should think about the terminology here. "Non-xonly" is totally fine to get this merged but I think it can be improved.